python/sglang/srt/managers/controller/tp_worker.py

```diff
@@ -355,14 +356,19 @@ def get_new_fill_batch(self) -> Optional[Batch]:
                     req.extend_input_len += delta
                     req.prefix_indices = req.prefix_indices[:-delta]
                     if req.image_offset is not None:
-                        req.image_offset += delta
+                        if isinstance(req.image_offse, list):
+                            req.image_offset  = [x + delta for x in req.image_offset]
+                        else:
+                            req.image_offset += delta
             if req.extend_input_len == 0 and req.max_new_tokens() > 0:
                 # Need at least one token to compute logits
                 req.extend_input_len = 1
                 req.prefix_indices = req.prefix_indices[:-1]
                 if req.image_offset is not None:
-                    req.image_offset += 1
-
+                    if isinstance(req.image_offset, list):
+                        req.image_offset = [x + 1 for x in req.image_offset]
+                    else:
+                        req.image_offset += 1
```

Comment:
This part is not needed anymore after some of our recent refactors https://github.com/sgl-project/sglang/pull/980.